expertise/execute_expertise.py

```diff
@@ -158,6 +158,56 @@ def execute_expertise(config):
                 scores_path=Path(config['model_params']['scores_path']).joinpath(config['name'] + '_sparse.csv')
             )
 
+    if 'submission_group' in config:
```

Comment:
I'd suggest moving this whole part into a function and using it inside the `all_scores` function for each model type.

Reply:
What if sparse scores are being computed between groups? Then this step will be missed - do you think I should also add it into the `sparse_scores` functions as well?

Also, this block of code requires access to the `dataset` field of the config, which is not passed to the models so I would have to add an additional argument to these functions. Though I do agree that this block should probably be in a function.